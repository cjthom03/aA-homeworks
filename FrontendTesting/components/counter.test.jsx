import React from 'react';
import { shallow } from 'enzyme';
import Counter from './counter';

describe("Counter", () => {
  it("starts with a count of 0", () => {
    let container = shallow(<Counter />);
    let counterCount = container.state().count;
    expect(counterCount).toBe(0);
  });
});
