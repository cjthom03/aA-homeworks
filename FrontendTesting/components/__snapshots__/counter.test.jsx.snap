// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter matches the initial snapshot 1`] = `
<div>
  <p>
    Count: 
    0
  </p>
  <button
    className="add1"
    onClick={[Function]}
  >
    Add 1
  </button>
</div>
`;

exports[`Counter matches the initial snapshot 2`] = `
<div>
  <p>
    Count: 
    2
  </p>
  <button
    className="add1"
    onClick={[Function]}
  >
    Add 1
  </button>
  <h1>
    FOO
  </h1>
  <h1>
    FOO
  </h1>
</div>
`;
